iwyu_tool_test.py

```diff
@@ -363,6 +363,34 @@ def test_extra_args_with_sep(self):
         self.assertEqual(['arg1', '--', 'another_arg1'],
                          self.main.call_args['extra_args'])
 
+    def test_jobs_normal(self):
+        """ Nonzero values for -j are forwarded as they are. """
+        argv = ['iwyu_tool.py', '-p', '.', '-j', '4']
+        iwyu_tool._bootstrap(argv)
+        self.assertEqual(4, self.main.call_args['jobs'])
+
+
+    def test_jobs_zero(self):
+        """ -j=0 will not hang the program. """
```

Comment:
Hmm, I don't think this testcase demonstrates `-j=0` doesn't hang, just that zero translates to some arbitrary non-zero?

Reply:
I guess I just change the comment?

Reply:
Yeah, that makes sense. I was trying to come up with a way of testing not hanging, but these tests are heavily mocked, so I don't think that's straightforward.